qhana_plugin_runner/util/plugins.py

```diff
@@ -270,3 +273,21 @@ def register_plugins(app: Flask):
                 plugin_blueprint,
                 url_prefix=f"{url_prefix}/plugins/{plugin.identifier}/",
             )
+
+
+def get_execution_options(url: str) -> dict:
```

Comment:
I would prefer this to be closer to the entity marshalling utilities (see added imports; possibly in a new module related to execution metadata)

Reply:
The function is now in [`qhana_plugin_runner.plugin_utils.metadata_utils`](https://github.com/UST-QuAntiL/qhana-plugin-runner/blob/ebf6fc369499720dcfb3f0a715d72904673e410a/qhana_plugin_runner/plugin_utils/metadata_utils.py#L21-L36).